### PR TITLE
disable member extension in role and group review

### PR DESCRIPTION
**Pull request by ArtjomsPorss**
# Description
disable member extension in role and group review when extension or review (days) are not set role/group in settings

# Contribution Checklist:
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have read the [contribution guidelines](https://github.com/AthenZ/athenz/blob/master/CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request.**

### CODE REVIEW

File: ui/src/components/denali/Alert.js [hunk 1]
```diff
@@ -226,6 +226,7 @@ class Alert extends React.Component {
                         <div
                             className='close-button'
                             onClick={this.props.onClose}
+                            data-wdio={'alert-close'}
                         >
                             <Icon
                                 color={rgba(colors.grey800, 0.4)}
```
Comment (ArtjomsPorss) [inline ui/src/components/denali/Alert.js:232]: to close confirmation tooltip in ui during functional tests and thus make test execution faster

File: ui/src/components/review/ReviewList.js [hunk 2]
```diff
@@ -109,6 +116,12 @@ class ReviewList extends React.Component {
                         _csrf={this.props._csrf}
                         onUpdateSuccess={this.submitSuccess}
                         justification={this.props.justification}
+                        expiryOrReviewSettingIsSet={
+                            0 <
+                            getSmallestExpiryOrReview(
+                                this.props.collectionDetails
+                            )
+                        }
                     />
```
Comment (ArtjomsPorss) [inline ui/src/components/review/ReviewList.js:125]: if expiry or rerview days in Group are present, then pass `true` here

File: ui/src/components/review/ReviewRow.js [hunk 3]
```diff
@@ -128,6 +130,7 @@ export default class ReviewRow extends React.Component {
                         value='extend'
                         checked={this.state.selectedOption === 'extend'}
                         onChange={this.onReview}
+                        disabled={!this.props.expiryOrReviewSettingIsSet}
                     />
```
Comment (ArtjomsPorss) [inline ui/src/components/review/ReviewRow.js:134]: if expiry or review days not set - disable Extend option

File: ui/src/components/review/ReviewRow.js [hunk 4]
```diff
@@ -44,7 +44,9 @@ export default class ReviewRow extends React.Component {
         super(props);
         this.api = this.props.api;
         this.onReview = this.onReview.bind(this);
-        let selectedOption = 'extend';
+        let selectedOption = this.props.expiryOrReviewSettingIsSet
+            ? 'extend'
+            : 'no-action';
         this.state = {
```
Comment (ArtjomsPorss) [inline ui/src/components/review/ReviewRow.js:50]: if expiry or review days set, then default option is extend, otherwise - no action

**Review by chandrasekhar1996 — APPROVED** [review]

**Pull request merged**